### PR TITLE
new memes + better doc

### PR DESCRIPTION
**Pull request by mashb1t**
Hey @paulborm,
here are some new memes for the soundboard, hope you enjoy them.

Ohhhhhh may be a bit too long if sb. decides to spam it as it get's loud during a few stacked playbacks. Maybe this one can be removed.

### CODE REVIEW

**Comment by paulborm** [comment]
Hi @mash1t 
can you please check the length and file size of your sounds and images again?  I know there are no guidelines but they should be as small as possbile.
You can achieve by:
- images: use jpg and compress the hell out of it using [imageoptim](https://imageoptim.com/mac) with the lowest possible quality setting and max width of around 500px. Goal should be a file under 10kb (it depends)
- sounds:  length between 1-2 seconds. Crop it if possible and compression (readme)[https://github.com/paulborm/soundboard#image-and-audio-compression]

Other changes:
- package.json is great
- .env is great
- not sure about the `now` part beause it's not necessary for development - maybe remove it until there is a real deployment :D

Thanks ❤️

**Comment by mashb1t** [comment]
@paulborm images have been resized to 512px width, sound files are now 48k (only ba-dum-tss.mp3 being 64k). 
It would be great if you could have another look at the changes, thank you.

File: README.md [hunk 1]
```diff
@@ -4,10 +4,43 @@
...
+now login
+```
+
+### Adding secret
+
+Run this command for local development:
+
+```bash
+now secrets add api-url localhost:3001
+```
```
Comment (paulborm) [inline README.md]: Please remove the now part

File: client/now.json [hunk 2]
```diff
@@ -1,7 +0,0 @@
-{
-  "build": {
-    "env": {
-      "REACT_APP_API_URL": "@api-url"
-    }
-  }
-}
```
Comment (paulborm) [inline client/now.json]: Please undo deletion of `now.json` file

**Review by paulborm — CHANGES_REQUESTED** [review]
Two last changes and we can merge

File: README.md (hunk 1 above)
Comment (mashb1t) [reply README.md]: done

File: client/now.json (hunk 2 above)
Comment (mashb1t) [reply client/now.json]: done

**Pull request merged**